docs/faq/edit-a-hip.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: edit-a-hip
+title: Propose Changes to Existing HIP in GitHub
+pagination_label: Edit an Existing HIP
+sidebar_label: Edit an Existing HIP
+description: Helium Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /edit-a-hip
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+While HIPs are in discussion, the actual text is included in the
+[Helium HIP Repository on GitHub](https://github.com/helium/hip).
+
+Before a HIP goes to a vote, it is likely that changes and revisions will need to be made to an
+existing proposal.
+
+:::tip Key Take Aways
+
+This page will walk users through the process of:
+
+- Creating a personal copy, or "fork", of the Helium HIP repository.
+- Making edits using the in-browser text editor.
+- Submitting these changes for review.
+
+:::
+
+---
+
+## Fork the Repo
+
+:::info What the Fork
+
+Rather than making edits to the main repository, create a personal copy by "forking" the repository
+to your GitHub account.
+
+:::
+
+1. Log in to an existing account or create a new account on [GitHub](https://github.com).
+2. Open the Helium HIP Repository at [https://github.com/helium/hip](https://github.com/helium/hip)
+3. Click the `Fork` button in the upper right-hand corner.
```

Comment:
```suggestion
3. Click the `Fork` button in the upper right-hand corner. This will create a full copy of the repository in your account as it exists at the current time.
```